pkg/providers/property/data_akamai_property_search.go

```diff
@@ -0,0 +1,115 @@
+package property
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+	"github.com/akamai/AkamaiOPEN-edgegrid-golang/v2/pkg/papi"
+	"github.com/akamai/terraform-provider-akamai/v2/pkg/akamai"
+	"github.com/akamai/terraform-provider-akamai/v2/pkg/tools"
+)
+
+func dataSourcePropertySearch() *schema.Resource {
```

Comment:
```suggestion
func dataSourcePropertiesSearch() *schema.Resource {
```
please rename property to properties everywhere in code and in documentation, as an example:
`akamai_property_search` -> `akamai_properties_search` and etc.

Reply:
renamed property to properties

Reply:
@Fuochi thank you for addressing it. I added two more suggestions on _properties_search.md_